core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -38,16 +38,21 @@
     </div>
   </div>
 
+
+
   <div class="advance-btn-wrapper">
-    <span class="advance-btn-description" ng-show="advBtnHovered">Add new question</span>
+    <div ng-show="chooseQuestionType" class="question-type-wrapper" >
+      <button type="button" class="oppia-question-button btn btn-primary" ng-click="chooseQuestionType = false; addNewQuestion('multiple');">Multiple Choice Input</button>
+    </div>
+    <span class="advance-btn-description" ng-show="advBtnHovered" ng-if="!chooseQuestionType">Add new question</span>
     <button ng-disabled="!canAddNewQuestion()" type="button"
-            class="btn btn-default advance-btn" ng-click="addNewQuestion(); advBtnHovered = false"
+            class="btn btn-default advance-btn" ng-click="chooseQuestionType = true; advBtnHovered = false"
```

Comment:
I think it would be better to set the `$scope` variable within a JS function, that way it's easier to keep track of down the road.